prosody/rootfs/etc/cont-init.d/10-config

```diff
@@ -26,6 +26,11 @@ if [[ "$(stat -c %U /prosody-plugins-custom)" != "prosody" ]]; then
 fi
 
 if [[ ! -f $PROSODY_CFG ]]; then
+    if [[ $ENABLE_XMPP_WEBSOCKET -eq 1 && -z "$PUBLIC_URL" ]]; then
```

Comment:
Let's drop this, PUBLIC_URL is now mandatory anyway. You can add something like this: https://github.com/jitsi/docker-jitsi-meet/blob/0c957941bc36795dccf241deaec18e23a004c279/jvb/rootfs/defaults/sip-communicator.properties#L29